frontend/package.json

```diff
@@ -24,48 +24,50 @@
     "@babel/core": "^7.0.0",
```

Comment:
did we upgrade all the packages? so all dependencies that we can update are in the most recent version?

Reply:
Yes, now I've upgraded all the packages to their latest versions, except those: https://codimd.puzzle.ch/tGhXgK1kTKimo0Cv473FJA#